lib/rex/proto/kerberos/pac/server_checksum.rb

```diff
@@ -12,13 +12,22 @@ class ServerChecksum < Element
           #   @return [Integer] The checksum type
           attr_accessor :checksum
 
+          def checksum_length
+            if checksum == Rex::Proto::Kerberos::Crypto::Checksum::SHA1_AES128 ||
```

Comment:
Can we throw a link to the spec here and in the other place that uses it?

https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-pac/6e95edd3-af93-41d4-8303-6c7955297315

Or we could introduce a shared constant which we can do a hash lookup on here instead of these if statements, and throw the link to the spec on that introduced constant 👀 

That should help stop folk fixing the size in one file and not in the other

Reply:
can do, I think I have what you're looking for over in my PAC bindata PR already, I just wasn't too concerned here since it'll be being replaced shortly but happy to do it
https://github.com/rapid7/metasploit-framework/pull/17209/files#diff-9294782a64293f5f3f5e78e1cbb79ccc54e885053b2da133d93f28fcceb34eabR37-R44

Reply:
Looks good to me; Will resolve this :+1: